datacenter/ucp/2.1/guides/user/services/index.md

```diff
@@ -7,12 +7,12 @@ title: Deploy an app from the UI
 
 With Docker Universal Control Plane you can deploy applications from the UI
 using `docker-compose.yml` files. In this example, we're going to deploy an
-application that allows users to vote on whether they prefer cats or dogs.
+application that allows users to vote on whether they prefer cats or dogs. 😺 🐶
```

Comment:
I see. All the benefits of enterprise-grade software, but with extra fluffiness.
Well played!